src/test/javascript/spec/component/shared/feedback/feedback-collapse.component.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { FeedbackCollapseComponent } from 'app/exercises/shared/feedback/collapse/feedback-collapse.component';
+
+describe('FeedbackCollapseComponent', () => {
+    /*
+     * Same value as in feedback-collapse.component.ts
+     */
+    const FEEDBACK_PREVIEW_CHARACTER_LIMIT = 300;
```

Comment:
This value should be at some central place. Duplicating it across the codebase doesn't seem like a good idea